src/main/java/com/google/javascript/gents/CollectModuleMetadata.java

```diff
@@ -180,6 +191,10 @@ void registerProvidesModule(Node n, String file, String namespace) {
     /** {@code True}, if the module has any imports (e.g.{@code goog.require}). */
     private boolean hasImports = false;
 
+    private String namespace = "";
+    /** Module has a default export. */
```

Comment:
Technically, prior to conversion there is no notion of 'default export' in goog.module system (its a purely ES6 concept). It would be more correct to say 'the module's clutz generated .d.ts will have a default export'

Reply:
Done.